spec/filters/grok_spec.rb

```diff
@@ -702,4 +702,66 @@
     end
   end
 
+  describe  "grok with nil coerced value" do
+    config <<-CONFIG
+      filter {
+        grok {
+          match => { "message" => "test (N/A|%{BASE10NUM:duration:float}ms)" }
+        }
+      }
+    CONFIG
+
+    sample "test 28.4ms" do
+      insist { subject["duration"] } == 28.4
+      insist { subject["tags"] }.nil?
+    end
+
+    sample "test N/A" do
+      insist { insist { subject.to_hash }.include?("duration") }.fails
```

Comment:
@jsvd actually, `include?` is buggy. I want to check if the key is in hash or not.. thats why the `to_hash`